index.js

```diff
@@ -0,0 +1,167 @@
+// - A button that displays the text "get pokemon!" and when clicked triggers the function "getPokemon"
+// - A button that displays the text "battle!" and when clicked triggers the function "battlePokemon"
+console.log("hey")
+document.addEventListener("DOMContentLoaded", () => {
+    // let form = document.querySelector("submit", (event))
+    // form.addEventListener("submit", (event => {
+    //     return event.preventDefault();
+
+    // }))
+    const getPokemon1 = async () => {
+      let res;
+      try {
+        let pokeId1 = Math.floor(Math.random() * 807)
+        res = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokeId1}`);
```

Comment:
Please remove code that you are not using. It's pretty confusing to read. You don't need 2 event listeners on your document listening for the same event ("DOMContentLoaded").